content/md/en/docs/reference/frame-macros.md

```diff
@@ -1,118 +1,206 @@
 ---
-title: Frame macros
+title: Runtime macros
```

Comment:
I think keeping the title FRAME is more accurate, since these macros are NOT required for every runtime development. just stuff that uses frame

Reply:
what is your take on why the title would change?

Reply:
I was thinking that the context of the discussion was about using the macros in a runtime, but the main motivation was to avoid using proper nouns and acronyms in headings so that headings focus on the topic at hand and not so much on Parity-specific or Substrate-specific terminology.